### ISSUE TITLE
[Windows terminal>Settings>Terminal chat]: Visual labels are not programmatically associated with edit fields; screen reader announces only “Edit”

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19694]
### Windows Terminal version

1.25.3571.0

### Windows build number

Windows 11 Version 26H1 (OS Build 28000.1340)

### Steps to reproduce

Repro Steps:
1.Launch the Window terminal app.
2.Navigate to the 'New Open Tab' and select the 'Settings' menu item.
3.Settings page will open.
4.Now navigate to 'Terminal chat' and select 'it'.
5.Terminal chat page will open.
6.Now turn on the narrator using 'CTRL+Win+Enter' keys.
7.Now navigate to 'Edit' fields present on the page.
8.Observe the issue here.

User Experience:
Screen reader users are unable to identify the purpose of form fields, which can lead to confusion and a higher rate of form completion errors. This does not meet fundamental accessibility standards for forms.

WCAG Reference Link:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Attachements:
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQAUrnj56gCAQZZkNuo26uuTAQ4RhZZLcXkiXI23ju8U5Qw?e=bYFpgJ

### Expected Behavior

Visual label or name should be programmatically bound to its corresponding edit field.

### Actual Behavior

All visual labels for edit fields are not programmatically associated with their respective input controls.
When navigating the form using a screen reader, each input field is announced only as “Edit” with no label or context.
This makes it difficult for screen reader users to understand the purpose of each edit field.

**Comment by carlos-zamora** [comment]
Looks like we need to add an `AutomationProperties.Name` to these text fields in the Terminal Chat page.

<img width="1913" height="519" alt="Image" src="https://github.com/user-attachments/assets/d9546b30-4b5b-43ef-be99-cd071cef01b3" />

https://github.com/user-attachments/assets/8dd83e56-4530-4cd7-908d-842f1895564e

**Comment by carlos-zamora** [comment]
Fixed by #19717
